layouts/shortcodes/release-data.html

```diff
@@ -33,10 +33,10 @@ <h3 id="release-v{{ $dataIdVersion }}">{{ $dataVersion }}</h3>
 </div>
 
 {{- end -}}
-<!-- not yet localized, mark as English -->
-<p lang="en">
-Complete {{ $dataVersion }} <a href="/releases/patch-releases/#{{ replace $dataVersion `.` `-` }}">Schedule</a>
-and <a href="https://git.k8s.io/kubernetes/CHANGELOG/CHANGELOG-{{ $dataVersion }}.md">Changelog</a>
+<p>
+{{ T "release_complete" }} {{ $dataVersion }}
+<a href="/releases/patch-releases/#{{ replace $dataVersion `.` `-` }}">{{ T "release_schedule" }}</a>{{ T "inline_list_separator" }}
```

Comment:
```suggestion
<a href="/releases/patch-releases/#{{ replace $dataVersion `.` `-` }}">{{ T "release_schedule" }}</a> {{ T "and" }}
```
For localization, should we preserve "and", like the original? Or does comma work better?

Reply:
It's a compromise. Another option is to make a format string that a localization team defines to build the sentence they need, with the `<a>` elements supplied as its input.

Let's do _something_ initially and then refine if we want to.